demo-app/cypress.config.js

```diff
@@ -0,0 +1,9 @@
+const { defineConfig } = require("cypress");
+
+module.exports = defineConfig({
+  e2e: {
+    setupNodeEvents(on, config) {
+      // implement node event listeners here
+    },
+  },
+});
```

Comment:
If you want to test just some components you will add next
```suggestion
module.exports = defineConfig({
  e2e: {
    setupNodeEvents(on, config) {
      // implement node event listeners here
    },
  },
    component: {
    devServer: {
      framework: "create-react-app",
      bundler: "webpack",
    },
  },
});
```